mlir/test/Dialect/Linalg/convolution/roundtrip-convolution.mlir

```diff
@@ -0,0 +1,124 @@
+// The following test examples of linalg convolution named ops lowered to linalg.generic and then
+// lifted back up to named op.
+// RUN: mlir-opt %s -linalg-generalize-named-ops | mlir-opt --linalg-specialize-generic-ops | FileCheck %s
+
+func.func @depthwise_conv_1d_nwc_wc(%input: memref<?x?x?xf32>, %filter: memref<?x?xf32>, %output: memref<?x?x?xf32>) {
+  linalg.depthwise_conv_1d_nwc_wc {dilations = dense<3> : tensor<1xi64>,
+                                       strides = dense<2> : tensor<1xi64>}
```

Comment:
[nit] This indentation is inconsistent with other indentations for attributes and also feels a bit arbitrary. Could I suggest moving attributes, consistently, to a dedicated line?
```suggestion
  linalg.depthwise_conv_1d_nwc_wc
        {dilations = dense<3> : tensor<1xi64>, strides = dense<2> : tensor<1xi64>}
```